### PR TITLE
feat: publish wells sdk [release]

### PR DESCRIPTION
**Pull request by Sollimann**
lets try again...

[SBS-2708] - Handle union of GeoJson and polygon wkt input for method getWellsByPolygon

You can now fetch wells by either a polygon [ WKT (well-known text)](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) or by [GeoJson](https://geojson.org/).

I have also made the package public (v0.0.1). @mrtraser does my package.json look right?

ps! had to create a new PR due to commit lint.

[SBS-2708]: https://cognitedata.atlassian.net/browse/SBS-2708

### CODE REVIEW

File: packages/wells/src/client/model/GeoJson.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+   * @type {string}
+   * @memberof Polygon
+   */
+  type: 'Polygon';
+  /**
+   * @type {GeometryPoint[]}
+   * @memberof Geometry
+   */
+  coordinates: Point2D[][];
+}
```
Comment (Sollimann) [inline packages/wells/src/client/model/GeoJson.ts]: @kiril-me I guess you could add this into the geospatial-sdk and I can remove it from the wells sdk

File: packages/wells/package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-  "private": true,
+  "private": false,
   "name": "@cognite/sdk-wells",
   "license": "Apache-2.0",
   "repository": "cognitedata/cognite-sdk-js",
   "homepage": "https://github.com/cognitedata/cognite-sdk-js#readme",
   "main": "dist/index.js",
   "types": "dist/src/index.d.js",
-  "version": "0.0.0",
+  "version": "0.0.1",
```
Comment (polomani) [inline packages/wells/package.json]: no need to do this, it'll happen automatically

**Comment by polomani** [comment]
fyi you don't need to re-open the pr just because of commitlint.
you can rename pr title on github and rerun ci

**Review by drsphelps — APPROVED** [review]
lgtm 👍

**Comment by Sollimann** [comment]
@cognitedata/devx-externals @cognitedata/core-api could I get your blessing before merging?

File: packages/wells/src/client/model/GeoJson.ts (hunk 1 above)
Comment (rek) [reply packages/wells/src/client/model/GeoJson.ts]: https://github.com/cognitedata/cognite-geospatial-sdk/pull/71/files

Comment (Sollimann) [reply packages/wells/src/client/model/GeoJson.ts]: Yea, I will update this sdk accordingly in a separate PR

**Review by hhenriques — APPROVED** [review]
@Sollimann looking good

**Pull request merged**